incubator/hnc/pkg/reconcilers/hierarchical_namespace_test.go

```diff
@@ -57,24 +56,14 @@ var _ = Describe("HierarchicalNamespace", func() {
 	})
 
 	It("should set the hns.status.state to Conflict if the namespace's owner annotation is wrong", func() {
-		// Create "bar" hns in "foo" namespace.
-		foo_hns_bar := newHierarchicalNamespace(barName, fooName)
-		updateHierarchicalNamespace(ctx, foo_hns_bar)
-
-		// It should set the self-serve subnamespace's owner annotation to the owner
-		// namespace (should set bar's owner annotation to "foo").
-		Eventually(func() string {
-			return getNamespace(ctx, barName).GetAnnotations()[api.AnnotationOwner]
-		}).Should(Equal(fooName))
-
-		// Todo fix this flacky test on changing the namespace annotation. See issue:
-		//	 https://github.com/kubernetes-sigs/multi-tenancy/issues/560
-		// Sleep 1 second to avoid updating the namespace instance too quickly.
-		time.Sleep(1 * time.Second)
-
-		// Clear the owner annotation and the HNS state should be set to "Conflict".
-		clearAnnotations(ctx, barName)
-		Eventually(getHNSState(ctx, fooName, barName)).Should(Equal(api.Conflict))
+		// Create "baz" namespace.
+		bazName := createNS(ctx, "baz")
+		// Create "baz" hns in "foo" namespace. Even "baz" namespace has already existed, we can
+		// still create the hns instance using HNC SA to bypass the webhook here. However, we
+		// should get conflict state on the newly created "baz" hns instance.
```

Comment:
I see that the end result here is that we end up with an api conflict. However, I'm unclear how this ties into the test name because I don't see anything in here about annotations anymore.